src/main/java/org/broadinstitute/dsm/db/dao/settings/InstanceSettingsDao.java

```diff
@@ -41,6 +35,17 @@
             "FROM instance_settings " +
             "WHERE ddp_instance_id = (SELECT ddp_instance_id FROM ddp_instance WHERE study_guid = ?)";
 
+    private static final String SQL_GET_HIDE_SAMPLES_TAB_BY_STUDY_GUID = "SELECT " +
+            "hide_samples_tab " +
+            "FROM instance_settings " +
+            "WHERE ddp_instance_id = (SELECT ddp_instance_id FROM ddp_instance WHERE study_guid = ?)";
```

Comment:
maybe remove that so that it is not getting used by accident?

Reply:
Actually I haven't added this SQL statement, it's been there, I just moved it down. 

As far as I see this statement is used in getHideSamplesTabByStudyGuid(String studyGuid) method which is does its' job properly, should we still remove this?

Or we can change this statement by changing the filtering logic of method.

e.g we can use `instance_name` instead of `study_guid` in filtering, let's wait for your answer and then we'll decide, thanks!

Reply:
if it is used in getHideSamplesTabByStudyGuid then let's leave it. sorry I missed the usage!